docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/DockerTemplateBase.java

```diff
@@ -45,7 +46,7 @@
     public final String hostname;
 
     public final String[] dnsHosts;
-    public final String[] volumes;
+    public final String volumesString;
```

Comment:
After analysing library changes it seems that better store input data as `String[]`. This should save us from changes in docker-java.